src/qibolab/instruments/zhinst.py

```diff
@@ -523,13 +506,11 @@ def run_exp(self):
         self.exp = self.session.compile(
             self.experiment, compiler_settings=COMPILER_SETTINGS
         )
-        # self.exp.save_compiled_experiment("saved_exp")
```

Comment:
This could be interesting to keep somewhere for debugging purposes.

Reply:
I wouldn't keep commented out lines in the code, for the purpose of uncommenting them whenever debugging is needed. I can add a dedicated function if you think this is usefull.